.eslintrc.json

```diff
@@ -46,7 +46,7 @@
     "camelcase": ["error", { "allow": ["^INTERNAL_"] }],
     "react/function-component-definition": [
       "error",
-      { "namedComponents": "arrow-function" }
+      { "namedComponents": "function-declaration" }
```

Comment:
just preference, happy to remove if you feel strongly

Reply:
I'd suggest split that into another PR. It introduces too many irrelevant changes.

Reply:
Sure no problem. Sorry, I couldn't resist 😅